src/modules/Error404/components/Error404/Error404.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { Component } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faExclamationTriangle } from '@fortawesome/free-solid-svg-icons';
+
+export default class Error404 extends Component{
+    render(){
+        return(
+            <div>
```

Comment:
You need to wrap the component content in `<Regular>...</Regual>`. So that the user can see header on the page.